cruise-control-metrics-reporter/src/test/java/com/linkedin/kafka/cruisecontrol/metricsreporter/utils/CCEmbeddedKRaftController.java

```diff
@@ -88,13 +84,13 @@ public String clusterId() {
 
     private Properties createControllerProperties() {
         Properties props = new Properties();
-        props.setProperty(KRaftConfigs.PROCESS_ROLES_CONFIG, "controller");
-        props.setProperty(KRaftConfigs.NODE_ID_CONFIG, String.valueOf(ID));
-        props.setProperty(KRaftConfigs.CONTROLLER_LISTENER_NAMES_CONFIG, "CONTROLLER");
-        props.setProperty(SocketServerConfigs.LISTENERS_CONFIG, "CONTROLLER://:" + _port);
-        props.setProperty(QuorumConfig.QUORUM_VOTERS_CONFIG, quorumVoters());
-        props.setProperty(ServerLogConfigs.LOG_DIR_CONFIG, _logDir.getAbsolutePath());
-        props.setProperty(KRaftConfigs.METADATA_LOG_DIR_CONFIG, _logDir.getAbsolutePath());
+        props.setProperty("process.roles", "controller");
```

Comment:
This is really smart to prevent dependencies on the Kafka libs.

Although I do think it'd be better to have them defined somewhere as constants.

E.g.,
``` java
    private static final String PROCESS_ROLE_PROP = "process.roles";
```

Reply:
It is actually a good idea, thanks! At this point, I do not want to reinvent where to store them, but I created constants for them inside the same class, with the same name that they had in kafka internal classes. From this point, it is easy to move those constants anywhere else.